src/hammer-tech/hammer_tech.py

```diff
@@ -646,7 +670,11 @@ def prepend_dir_path(self, path: str, lib: Optional[Library] = None) -> str:
         assert len(path) > 0, "path must not be empty"
 
         # If the path is an absolute path, return it as-is.
-        if path[0] == "/":
+        if path[0] == os.sep:
```

Comment:
Does `os.path.isabs()` work for this purpose?

Reply:
Ah, it does, good point.